src/dfcx_scrapi/tools_ml/utterance_generator_utils.py

```diff
@@ -195,46 +238,59 @@ def _generate_phrases(self, training_phrases: pd.DataFrame, set_size: int):
     def create_synthetic_set(
```

Comment:
@Hgithubacct same here with `set` and the method name as well as the input arg names.  

Perhaps `create_synthetic_dataset` is easiest.